convert2rhel/unit_tests/main_test.py

```diff
@@ -808,3 +808,20 @@ def test_handle_inhibitors_found_exception(monkeypatch, rollback_failures, retur
     ret = main._handle_inhibitors_found_exception()
 
     assert ret == return_code
+
+
+def test_confirm_user_backup(monkeypatch, caplog):
+    ask_to_continue_mock = mock.Mock()
+
+    monkeypatch.setattr(utils, "ask_to_continue", ask_to_continue_mock)
+
+    main.confirm_user_backup()
+
+    message = (
+        "Convert2RHEL modifies the systems during the analysis and then rolls back these "
+        "changes when the analysis is complete. In rare cases, this rollback can fail. "
+        "By continuing, you confirm that you have made a system backup and verified that "
+        "you can restore from the backup."
+    )
+
+    assert ask_to_continue_mock.call_count == 1
```

Comment:
You forgot to also assert with caplog that the message was printed.
```suggestion
    assert message in caplog.records[-1].message
    assert ask_to_continue_mock.call_count == 1
```

Reply:
Gotcha thank you for showing this to me, I will go ahead and push this up!